pyproject.toml

```diff
@@ -52,6 +53,42 @@ exclude = '''
                      # the root of the project
 )
 '''
+
+[tool.pylint.master]
+# Include the pylint_django plugin to avoid spurious warnings about Django patterns
+load-plugins="pylint_django"
+
+# Don't cache data for later comparisons
+persistent="no"
+
+[tool.pylint.basic]
+# No docstrings required for private methods (Pylint default), or for test_ functions, or for inner Meta classes.
+no-docstring-rgx="^(_|test_|Meta$)"
+# Allow "ot" (OnboardingTask instance) as an acceptable variable name
+good-names="ot,"
+
+[tool.pylint.messages_control]
+# Line length is enforced by Black, so pylint doesn't need to check it.
+# Pylint and Black disagree about how to format multi-line arrays; Black wins.
+# Disabled due to noise: too-few-public-methods, too-many-ancestors, too-many-instance-attributes
+disable = """,
+    line-too-long,
+    bad-continuation,
+    too-few-public-methods,
+    too-many-ancestors,
+    too-many-instance-attributes,
+    duplicate-code,
+    raise-missing-from,
+    super-with-arguments
```

Comment:
Do we have an issue open for fixing the `raise-missing-from` and `super-with-arguments` items?

Reply:
https://github.com/networktocode/ntc-netbox-plugin-onboarding/issues/115